bin/create-test-release.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
```

Comment:
Let's enable the so-called ["strict mode"](http://redsymbol.net/articles/unofficial-bash-strict-mode/)

```bash
set -euo pipefail
IFS=$'\n\t'
```